docker-compose.yml

```diff
@@ -1,54 +1,66 @@
 version: '3'
 services:
 
-    broker:
+    rabbitmq:
       container_name: rabbitmq
       image: 'rabbitmq:3-management'
+      healthcheck:
+        test: rabbitmq-diagnostics -q ping
+        interval: 30s
+        timeout: 30s
+        retries: 3
 
-    nginx:
-        container_name: nginx
-        build:
-          context: ./deployment/nginx/
-        restart: always
-        ports:
-        - "8080:80"
-        # - "80:80"
-        volumes:
-          - djam_static:/djam_static
-        # - ./log:/var/log/nginx
-        depends_on:
-          - server
-
-    server:
+    djam:
         container_name: djam
         build:
           context: ./
         restart: always
         env_file:
-          - .env
+          - .env_docker
 #        command: uwsgi  --emperor uwsgi.ini
```

Comment:
if we don't use uwdgi let's remove this comment. Otheriwse we should replace with guvicorn or any other command we shall run.